concrete/src/Console/Command/RefreshEntitiesCommand.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Concrete\Core\Console\Command;
+
+use Concrete\Core\Console\Command;
+use Concrete\Core\Database\DatabaseStructureManager;
+use Concrete\Core\Package\Event\PackageEntities;
+use Concrete\Core\Support\Facade\Application;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+
+class RefreshEntitiesCommand extends Command
+{
+    protected function configure()
+    {
+        $errExitCode = static::RETURN_CODE_ON_FAILURE;
+
+        $this
+            ->setName('c5:entities:refresh')
+            ->setDescription('Refresh the Doctrine database entities')
+            ->addEnvOption()
+            ->setCanRunAsRoot(false)
+            ->setHelp(<<<EOT
+This command will refresh the Doctrine database entities and set the following return codes
+
+  0 operation completed succesfully
+  {$errExitCode} errors occurred
+EOT
+            )
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $app = Application::getFacadeApplication();
+
+        $pev = new PackageEntities();
+        $app->make(EventDispatcherInterface::class)->dispatch('on_refresh_package_entities', $pev);
```

Comment:
@mlocati is there a reason why you've called the event `on_refresh_package_entities`?

Reply:
Yes, because [it's already named so](https://github.com/concrete5/concrete5/blob/8.4.1/concrete/controllers/single_page/dashboard/system/environment/entities.php#L42) :wink:.
BTW don't ask why I named it like that in #6045 (I can't remember what I ate for dinner yesterday🤣)